LayoutTests/platform/glib/TestExpectations

```diff
@@ -1183,6 +1183,9 @@ webkit.org/b/213699 http/wpt/mediarecorder/MediaRecorder-audio-bitrate.html [ Pa
 webkit.org/b/213699 http/wpt/mediarecorder/MediaRecorder-AV-audio-video-dataavailable.html [ Crash Timeout Failure ]
 webkit.org/b/213699 http/wpt/mediarecorder/MediaRecorder-first-frame.html [ Failure Pass ]
 
+# No support for Matroska container in GStreamer port.
```

Comment:
WebM is supported but only for the vp8 and vorbis codecs, afaict.

Reply:
> WebM is supported but only for the vp8 and vorbis codecs, afaict.

yes indeed, and that is all that should be supported per spec.

Unfortunately Google has decided to not even comply to the standard they designed and they allow h264/pcm in a webm.

Worse, that is their default format used by MediaRecorder and they will play any webm as a mkv even if invalid see https://groups.google.com/a/webmproject.org/g/webm-discuss/c/27DMi7D7Gsg?pli=1